css/index.css

```diff
@@ -1,3 +1,529 @@
+	  border: 1px solid black;
+	  padding: 20px;
+	  background: lightgray;
+	  display: flex;
+	  flex-direction: column;
+	  justify-content: space-evenly;
+  }
+
+  .service .con p {
+	line-height: 1.2;
+	padding-bottom: 20px;
+  }
+
+  .service .button {
+	  border: 1px solid black;
+	  border-radius: 5px;
+	  width: 200px;
+	  height: 40px;
+	  font-size: 13px;
+	  font-weight: bold;
+	  color:black;
+	  text-decoration: none;
+	  background: white;
+	  display: flex;
+	  justify-content: center;
+	  align-items: center;
+  }
+
+  .additional-info {
+	  display: flex;
+	  justify-content: space-between;
+	  margin-bottom: 100px;
+  }
+
+  .facts , .services-info-img {
+	  width: 48%;
+  }
+
+  .facts {
+	  display: flex;
+	  flex-direction: column;
+	  justify-content: flex-start;
+  }
+
+  .facts p {
+	margin-bottom: 20px;
+  }
+
+  .facts ul {
+	display: block;
+	line-height: 2;
+	font-weight: bold;
+  }
+
+  .facts li {
+	list-style-type: square;
+	font-size: 15px;
+	list-style-position: inside;
+  }
+
+
+	@media screen and (max-width: 800px){
+
+		.container {
+			width: 95%;
+			/* TB LR */
+			margin: 0 auto;
+			}
+			
+			header {
+				display: flex;
+				flex-direction: column;
+				align-items: center;
+			}
+
+			header nav {
+				width: 100%;
+				padding-left: 20px;
+				padding-right: 20px;
+			}
+
+			header .logo {
+				order: -1;
+				}
+
+					  /* cta */
+			.cta {
+				display: flex;
+				flex-direction: column;
+			}
+
+			.cta .cta-text {
+				/* display: inline-block; */
+				vertical-align: top;
+				font-size: 77px;
+				margin-top: 60px;
+				text-align: center;
+				padding: 0px 20px;
+				/* border: 1px solid red; */
+				}
+				
+				.cta .cta-text button {
+				background: white;
+				vertical-align: top;
+				margin-top: 6px;
+				padding: 8px 44px;
+				font-size: 17px;
+				border: 1px solid black;
+				cursor: pointer;
+				}
+				
+				.cta .cta-text button:hover {
+				background: black;
+				color: white;
+				}
+				
+				.cta img {
+					display: none;
+				}
+
+				.main-content .middle-img {
+					width: 100%;
+					
+					/* align-items: center; */
+					/* border: 1px solid red; */
+					}
+				
+	}
+
+
+@media screen and (max-width: 500px){
+
+
+	  /* header */
+	  header{
+		display: flex;
+		flex-direction: column;
+		justify-content: space-between;
+		align-items: center;
+	  }
+	
+	
+	  header nav {
+		display: flex;
+		justify-content: space-between;
+		flex-direction: column;
+		width: 100%;
+		/* height: 350px; */
+		/* border: 1px solid red; */
+		/* margin-top: 35px; */
+		align-items: center;
+		padding-bottom: 0px;
+		/* border-bottom: 1px solid black; */
+		margin: 0;
+	  }
+	  
+	  header nav a {
+			/* width: 100%;
+			height: auto; */
+			padding: 30px 100%; 
+			/* border-top: 1px solid grey; */
+			border-bottom: 1px solid grey;
+			font-size: 14px;
+			color: gray;
+			text-decoration: none;
+
+	  }
+	  
+	  header .logo {
+		margin: 0;
+		padding: 30px 100%;
```

Comment:
This line (`padding: 30px 100%`) seems to have some weird behavior. In Firefox (61), your logo seems to be off of the page at mobile view. In Chrome, this seems to work like a charm! I'd look into trying to figure out another way of doing this, if you're able, for the sake of supporting more browsers (and FF definitely isn't unmodern).